editor/block-toolbar/index.js

```diff
@@ -78,6 +79,7 @@ class BlockToolbar extends Component {
 							<BlockInspectorButton small />
 							<BlockModeToggle uid={ block.uid } small />
 							<BlockDeleteButton uids={ [ block.uid ] } small />
+							<UnknownConverter uid={ block.uid } small />
```

Comment:
Can we put it before Delete button? This is how it looks at the moment:

<img width="381" alt="screen shot 2017-10-31 at 15 30 27" src="https://user-images.githubusercontent.com/699132/32229151-76ef6618-be50-11e7-89e6-1adaad82bcd6.png">

@jasmussen @karmatosed, what would be the best place to put it?

Reply:
Sorry, yeah, should be the same as normal drop down. :)